tests/passing-configs/index.js

```diff
@@ -14,8 +15,6 @@ export function filePathToConfigObj(filepath) {
   return {
     config,
     filepath,
-    name: path.parse(filepath).name,
+    name: pathParse(filepath).name
```

Comment:
Huh. Interesting. Oh well `¯\_(ツ)_/¯`, thanks for working on this!

If you'd like to update it that'd be great. Otherwise I'd be happy to merge it as it is and fix it later :-) Up to you.